addons/html_editor/static/tests/html_field.test.js

```diff
@@ -1047,6 +1056,97 @@ test("'Video Link' command is available", async () => {
     expect(queryAllTexts(".o-we-command-name")).toEqual(["Video Link"]);
 });
 
+test.tags("desktop");
+test("should display overlay on video hover and handle video replacement and removal correctly", async () => {
+    await mountView({
+        type: "form",
+        resId: 1,
+        resModel: "partner",
+        arch: `
+            <form>
+                <field name="txt" widget="html"/>
+            </form>`,
+    });
+    setSelectionInHtmlField();
+
+    let callCount = 0;
+    await onRpc("/web_editor/video_url/data", async () => {
+        callCount++;
```

Comment:
Don't use a counter, but use the actual request param (`video_url`) like this:
to return the adequate result:
```suggestion
    await onRpc("/web_editor/video_url/data", async (request) => {
        const videoUrl = (await request.json()).params.video_url;
```